scout/adapter/mongo/variant.py

```diff
@@ -257,6 +257,24 @@ def get_causatives(self, institute_id):
         ])
         return [item['_id'] for item in query]
 
+    def get_case_causatives(self, case_id):
```

Comment:
There is a function called `get_causatives` in adapter.variant. Please add the argument `case_id=None` to that function and add the logic there